tutorials/fundamentals/concat-string-array.md

```diff
@@ -0,0 +1,17 @@
+JavaScript's `join()` method is handy for turning elements in
+an array into a string. You can specify what you want the
+elements to be joined by, like  `-`, for example but it defaults to a space.
+The downside about being able to concatenate a JavaScript array is that
```

Comment:
This needs to be a lot shorter and a lot more clear. Let's do "JavaScript arrays can contain values of different types. If you only want to concatenate strings, you can filter out non-string values using [`filter()`](https://masteringjs.io/tutorials/fundamentals/filter) and [`typeof`](https://masteringjs.io/tutorials/fundamentals/typeof)"